src/features/click-to-load.js

```diff
@@ -615,14 +628,14 @@ function createPlaceholderElementAndReplace (widget, trackingElement) {
 
     // YouTube
     if (widget.replaceSettings.type === 'youtube-video') {
-        sendMessage('updateYouTubeCTLAddedFlag', true)
+        lazy.messaging.notify('updateYouTubeCTLAddedFlag', true)
```

Comment:
You cannot send primitive values like bools/string etc in the new system. It must be a `key: value` pair. This restriction helps with documentation, but more importantly it gives the value a name/label that native sides can use to refer to it.

With all _new_ features, you'd describe the message payloads in a way that can contribute to the auto-generated documentation, so it's a good habit to get into. To fix for your PR, all calls to `notify` or `request` would change to look like

```js
lazy.messaging.notify('updateYouTubeCTLAddedFlag', { addedFlag: true })

// or a generic 'payload'

lazy.messaging.notify('updateYouTubeCTLAddedFlag', { payload: true })
```

You can handle this in the custom transport if you don't want the Extension or Android to incur the cost of changing it - we don't need native changes. It's just the API restrictions are there to enforce certain practices :)